### PR TITLE
Handle UNC Paths

### PR DESCRIPTION
**Pull request by ClaudioESSilva**
Make possible read/write from/to UNC paths.
Also address issue: https://github.com/Microsoft/ReportingServicesTools/issues/33

### CODE REVIEW

File: ReportingServicesTools/Functions/CatalogItems/Write-RsCatalogItem.ps1 [hunk 1]
```diff
@@ -76,12 +76,25 @@ function Write-RsCatalogItem
...
+                $EntirePath = ConvertTo-UNCFilePath $item
+
+                if (!(Test-Path $EntirePath))
+                {
+                    throw "No item found at the specified path: $EntirePath!"
+                }
+            }
+            else
+            {
+                if (!(Test-Path $item))
```
Comment (parthsha) [inline ReportingServicesTools/Functions/CatalogItems/Write-RsCatalogItem.ps1]: What if you do only 1 Test-Path check and in if/else determine which value you want to use for $EntirePath?

File: ReportingServicesTools/Functions/Common/ConvertToUNC.ps1 [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function ConvertTo-UNCPath
```
Comment (parthsha) [inline ReportingServicesTools/Functions/Common/ConvertToUNC.ps1]: I was wondering does Convert-Path handle both regular paths as well as UNC paths? https://technet.microsoft.com/en-us/library/ee156816.aspx

**Comment by ClaudioESSilva** [comment]
Thanks for the review @parthsha !
I have rollback the change and just changed to the `Convert-Path` cmdlet. From my tests it works as expected.

File: ReportingServicesTools/Functions/CatalogItems/Write-RsCatalogItem.ps1 [hunk 3]
```diff
@@ -137,7 +136,11 @@ function Write-RsCatalogItem
                     $warnings = $null
                     try
                     {
+                    Write-Verbose $EntirePath
```
Comment (jtarquino) [inline ReportingServicesTools/Functions/CatalogItems/Write-RsCatalogItem.ps1]: can you fix the indentation

Comment (ClaudioESSilva) [reply ReportingServicesTools/Functions/CatalogItems/Write-RsCatalogItem.ps1]: Done. Thanks

**Pull request merged**